backend/accounts/baseaccount.go

```diff
@@ -133,6 +137,24 @@ func (account *BaseAccount) Notes() *notes.Notes {
 	return account.notes
 }
 
+// ProposeTxNote implements accounts.Account.
+func (account *BaseAccount) ProposeTxNote(note string) {
+	account.proposedTxNoteMu.Lock()
+	defer account.proposedTxNoteMu.Unlock()
+
+	account.proposedTxNote = note
+}
+
+// GetAndClearProposedTxNote returns the note previously set using ProposeTxNote(). If none was set,
+// the empty string is returned. The proposed note is cleared by calling this function.
+func (account *BaseAccount) GetAndClearProposedTxNote() (string, error) {
```

Comment:
Just string, no error? Simpler. No downsides, or so it seems.

Reply:
Ah yes, leftover from a previous version. fixed